### PR TITLE
feat(icon): show a red dot when we have unread notifications

### LINKED ISSUE #14788: Display a Red Dot on the tray icon of the app when there are unread messages
### Description

Part of [#10466](https://github.com/status-im/status-desktop/issues/10466)

First step of the Epic. Just start with a red dot when there are any types of unread messages. No need to bother with mentions or having a count for now.

### Acceptance Criteria

- when the app has an unread message, the app icon in the tray has a red dot

image

### PR DESCRIPTION
**Pull request by jrainville**
Fixes #14788

Adds a red dot on the tray icon if there is an unread message in an unmuted channel or in the activity center

[red-dot.webm](https://github.com/status-im/status-desktop/assets/11926403/9a9175f7-5cde-4dac-952b-a01c5dbefca7)

### CODE REVIEW

**Review by alexjba — APPROVED** [review]
Nice!

File: ui/app/mainui/StatusTrayIcon.qml [hunk 1]
```diff
@@ -6,13 +6,16 @@ SystemTrayIcon {
...
 
     property bool isProduction: true
+    property bool showRedDot: false
 
     signal activateApp()
 
     visible: true
+
+
     icon.source: Qt.platform.os === Constants.windows // TODO: Add status-logo-white with stroke for windows
```
Comment (caybro) [inline ui/app/mainui/StatusTrayIcon.qml:16]: Hmm so this is not gonna work for Windows... what was the problem there @alexjba ?

**Review by caybro — APPROVED** [review]
Nice! :+1:

**Review by MishkaRogachev — APPROVED** [review]
I like this dot

File: ui/app/mainui/StatusTrayIcon.qml (hunk 1 above)
Comment (alexjba) [reply ui/app/mainui/StatusTrayIcon.qml:16]: We didn't had proper assets for windows
https://github.com/status-im/status-desktop/issues/14855

Comment (alexjba) [reply ui/app/mainui/StatusTrayIcon.qml:16]: Alisher provided the proper tray icon with and without the red dot in this task https://github.com/status-im/status-desktop/issues/14855

Comment (jrainville) [reply ui/app/mainui/StatusTrayIcon.qml:16]: Oh nice. I can update the Windows icon in this PR then. Do you know if I can find the SVG somewhere?

Comment (jrainville) [reply ui/app/mainui/StatusTrayIcon.qml:16]: Ah actually, it uses png for Windows. I can just use those then!

**Comment by jrainville** [comment]
Alright, I finally made the Windows icons work. Turns out pngs can work, but I was using the wrong folder 🤦 
Anyway, sngs and lighter and cleaner, so win-win.

Here what it looks like on Windows:
image
image

Also, I fixed an issue where the unread section calculation also checked the profile for example, so if you had the backup seed phrase notification, you'd have the red dot. Now it only checks chat sections.
We can add wallet later if they have notifications too.

**Comment by jrainville** [comment]
@anastasiyaig @Valentina1133 I have this weird issue where the e2e tests appear stuck. When going to Jenkins, they seem to have passed, but failed on generating the report. I tried restarting twice and it's the same.

Do you think it's safe for me to merge still?

**Comment by Valentina1133** [comment]
> @anastasiyaig @Valentina1133 I have this weird issue where the e2e tests appear stuck. When going to Jenkins, they seem to have passed, but failed on generating the report. I tried restarting twice and it's the same.
> 
> Do you think it's safe for me to merge still?

I reran tests once again and now all good and green :)

**Pull request merged**